resources/qml/WelcomePages/WelcomeDialog.qml

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2019 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+import QtQuick.Window 2.2
+
+import UM 1.3 as UM
+import Cura 1.1 as Cura
+
+
+Window
+{
+    UM.I18nCatalog { id: catalog; name: "cura" }
+
+    title: catalog.i18nc("@title", "Welcome to Ultimaker Cura")
+    modality: Qt.ApplicationModal
+    flags: Qt.Window | Qt.FramelessWindowHint
+
+    width: 580  // TODO
+    height: 600  // TODO
```

Comment:
TODOs here.

Reply:
Added `screenScaleFactor` in f6d212dc8839161430481b7de4c59fca147e0b10